test/OpenTelemetry.Tests/Metrics/MetricsTest.cs

```diff
@@ -20,7 +20,7 @@
 using OpenTelemetry.Trace;
 using Xunit;
 
-namespace OpenTelemetry.Metrics.Test
+namespace OpenTelemetry.Metrics.Tests
```

Comment:
should that be changed to OpenTelemetry.Tests.Metrics?

Reply:
Probably not, as it would require extra `using OpenTelemetry.Metrics;`.